<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>922</y>
     <width>210</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-image : url(icon/backb.png); 
   border : 0px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="video1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>425</width>
     <height>530</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/feed4.png</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>670</y>
     <width>425</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 백제 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text2">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>670</y>
     <width>425</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 백제 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text3">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>670</y>
     <width>425</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 백제 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text4">
   <property name="geometry">
    <rect>
     <x>1442</x>
     <y>670</y>
     <width>425</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 백제 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="video2">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>130</y>
     <width>425</width>
     <height>530</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="video3">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>130</y>
     <width>425</width>
     <height>530</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="video4">
   <property name="geometry">
    <rect>
     <x>1442</x>
     <y>130</y>
     <width>425</width>
     <height>530</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backButton</zorder>
  <zorder>video1</zorder>
  <zorder>text1</zorder>
  <zorder>text2</zorder>
  <zorder>text3</zorder>
  <zorder>text4</zorder>
  <zorder>video2</zorder>
  <zorder>video3</zorder>
  <zorder>video4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
